CompuCell3D/CMakeLists.txt

```diff
@@ -148,10 +159,26 @@ if(BUILD_STANDALONE)
     set(SITE_PACKAGES_INSTALL_REL lib/site-packages)
     set(COMPUCELL3D_INSTALL_SCRIPT_DIR_FULL_PATH ${CMAKE_INSTALL_PREFIX})
     set(COMPUCELL3D_INSTALL_SCRIPT_DIR .)
+    SET(COMPUCELL3D_RUNTIME_DESTINATION bin)
+    SET(COMPUCELL3D_LIBRARY_DESTINATION lib)
+    SET(COMPUCELL3D_ARCHIVE_DESTINATION lib)
 else()
     file(RELATIVE_PATH SITE_PACKAGES_INSTALL_REL ${CMAKE_INSTALL_PREFIX} ${Python3_SITELIB})
     set(COMPUCELL3D_INSTALL_SCRIPT_DIR_FULL_PATH ${CMAKE_INSTALL_PREFIX}/bin)
     file(RELATIVE_PATH COMPUCELL3D_INSTALL_SCRIPT_DIR ${CMAKE_INSTALL_PREFIX} ${COMPUCELL3D_INSTALL_SCRIPT_DIR_FULL_PATH})
+
+    # note this only works with same layouts as used in miniconda. For example if Python is scattered all over the place this might not work properly
+    if(${CMAKE_SYSTEM_NAME} STREQUAL Windows)
+        SET(COMPUCELL3D_RUNTIME_DESTINATION ${PYTHON_BASE_DIR}/Library/bin)
+        SET(COMPUCELL3D_LIBRARY_DESTINATION ${PYTHON_BASE_DIR}/Library/bin)
+        SET(COMPUCELL3D_ARCHIVE_DESTINATION ${PYTHON_BASE_DIR}/Library/bin)
+    else(${CMAKE_SYSTEM_NAME} STREQUAL Windows)
+        SET(COMPUCELL3D_RUNTIME_DESTINATION ${PYTHON_BASE_DIR}/lib)
+        SET(COMPUCELL3D_LIBRARY_DESTINATION ${PYTHON_BASE_DIR}/lib)
+        SET(COMPUCELL3D_ARCHIVE_DESTINATION ${PYTHON_BASE_DIR}/lib)
+
+    endif()
```

Comment:
Same as above. Remove the redundant condition and add quotes.

Reply:
Fixed